### ISSUE TITLE
[BUG] Invalid fingerprint for PGP 

### ISSUE THREAD

**Issue by peternied** [opensearch-project/project-website#332]
**Describe the bug**
Invalid fingerprint for PGP 

**To Reproduce**
1. Go to https://opensearch.org/verify-signatures.html
2. See the text `Our current PGP key fingerprint is C2EE 2AF6 542C 03B4`

The key we publish is a subkey of the master public key, so this is technically incorrect.

The fingerprint is the subkey id, which you can verify with these commands:
```
$ gpg --list-keys --with-subkey-fingerprints C2EE2AF6542C03B4
pub   rsa4096 2021-05-11 [SC]
      C5B7498965EFD1C2924BA9D539D319879310D3FC
uid           [ unknown] OpenSearch project <opensearch@amazon.com>
sub   rsa2048 2021-05-11 [S] [expires: 2022-05-11]
      21873199B1030FCD49DA83F8C2EE2AF6542C03B4

```

**Expected behavior**
We could probably list both the fingerprint C5B7498965EFD1C2924BA9D539D319879310D3FC alongside the subkey id C2EE2AF6542C03B4.

_Thanks to @pashtet04 and @peterzhuamazon who provided context for this issue_

**Comment by stockholmux** [comment]
@peternied Would we just need to add the subkey? Something along the lines of:

`Our current PGP key fingerprint is C5B7498965EFD1C2924BA9D539D319879310D3FC and the subkey id is C2EE2AF6542C03B4`?

**Comment by peterzhuamazon** [comment]
@stockholmux the key we currently publish IS the subkey.
The `fingerprint` we published is the subkey ID.

These are all correct information on the website.

The confusing part is the fingerprint normally means the master public key fingerprint.
When you try to use GPG to verify fingerprint it will normally print out the master public key fingerprint.

However, if you use the command I provide above in the description, it will verify correctly that our subkey is indeed part of the master public key.

I think the solution here is to remove the subkey id and just post the master public key fingerprint.
Since the subkey id is already included in the 1st line of the key we published.

Thanks.
